### PR TITLE
Merge and adds Experimenter Menu

### PR DESCRIPTION
**Pull request by NicolajRJensen**
#3

### CODE REVIEW

File: src/scivu/scivu/ViewModels/ExperimenterMenuViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+using System.Windows.Input;
+using ReactiveUI;
+using scivu.Models;
+
+namespace scivu.ViewModels;
+
+public class ExperimenterMenuViewModel : ViewModelBase
+{
+    private readonly IReadSurvey _survey;
```
Comment (jonasreholt) [inline src/scivu/scivu/ViewModels/ExperimenterMenuViewModel.cs:9]: Fairly certain this should be `IObservable<IReadSurvey> _survey`.
At least if we want notification in the UI if anything change?

File: src/scivu/scivu/Views/ExperimenterMenuView.axaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <Button x:CompileBindings="False"
+                Command="{Binding $parent[Window].DataContext.Change}"
+                CommandParameter="StartSurvey">
+            Start Survey
+        </Button>
+                <Button x:CompileBindings="False"
+                Command="{Binding $parent[Window].DataContext.Change}"
+                CommandParameter="ExportData">
+            Export Data
+        </Button>
```
Comment (jonasreholt) [inline src/scivu/scivu/Views/ExperimenterMenuView.axaml:23]: Do we want to handle calling changing view using this syntax, or only using delegates in the code?
I find the delegates more straightforward to understand

File: src/scivu/scivu/ViewModels/ExperimenterMenuViewModel.cs (hunk 1 above)
Comment (NicolajRJensen) [reply src/scivu/scivu/ViewModels/ExperimenterMenuViewModel.cs:9]: Not relevant

File: src/scivu/scivu/Views/ExperimenterMenuView.axaml (hunk 2 above)
Comment (NicolajRJensen) [reply src/scivu/scivu/Views/ExperimenterMenuView.axaml]: Changed according to this comment

**Review by jonasreholt — APPROVED** [review]
LGTM

**Pull request merged**